modules/contact/src/ContactMaster.C

```diff
@@ -417,6 +417,9 @@ contactFormulation(const std::string & the_name)
   else if ("augmented_lagrange" == name)
     formulation = CF_AUGMENTED_LAGRANGE;
 
+  else if ("tangential_penalty" == name)
```

Comment:
> The compiler warns about if (var = constant) anyways...

Not all compilers warn about this under all combinations of warning flags.  In fact, it used to be a fairly common (albeit awful IMO) idiom in C programming to do `if (foo=malloc(bar))` because it would automatically test the return value.


> So this is just my general opinion, but if there are two ways of doing a thing we should pick one (that is actually what a style guide is all about!)

My opinion is this isn't a big enough thing to have a style guideline about and/or change every instance of, then again we enforce having spaces around operators and between `if` and `for`-statements... so it's admittedly a very subjective opinion.